frontend/src/components/common/navigation/FormBreadcrumb.tsx

```diff
@@ -27,7 +27,7 @@ const FormBreadcrumb = ({
   validPage,
 }: FormBreadcrumbProps): React.ReactElement => {
   return (
-    <Breadcrumb color="grey.300" separator={<ChevronRightIcon />} spacing="8px">
+    <Breadcrumb color="grey.300" separator={<ChevronRightIcon />} spacing="2">
```

Comment:
Nit (optional): when using chakra-sizes it can be nice to use numbers explicitly rather than strings to indicate the difference between the two types of sizes.